lib/core/test/unit/Cardano/Pool/DB/Properties.hs

```diff
@@ -1513,16 +1511,15 @@ prop_delistPoolsPersists DBLayer {..} cert =
         let poolid = view #poolId . snd $ cert
         -- delist pool
         run $ atomically $ delistPools [poolid]
-        delisted <- run . atomically . readPoolRegistration $ poolid
```

Comment:
This whole test doesn't make sense anymore. Because the delisted flag doesn't exist in the table in question. So it doesn't test anything.

Reply:
I agree that we could delete this test.

Though given your [earlier comment](https://github.com/input-output-hk/cardano-wallet/pull/2277#discussion_r513384085) about completely overwriting the set each time we do a `delistPools`, I think it would make sense to update the test so that it checks that the set is replaced, rather than augmented.

Reply:
Sure, but that's a different property. This test was about "persistence in light of certificate insertion".

Reply:
> Sure, but that's a different property. This test was about "persistence in light of certificate insertion".

I've now deleted this test.

Fixed in c94d57890fd9ce1099e06731a0e66f2bce6bde63.